code/Gamemodes/Gamemode.cs

```diff
@@ -74,14 +80,12 @@ internal virtual void Initialize()
 
 	internal virtual void Start() { }
 
-	internal virtual void OnClientJoined( IClient client )
-	{
-		PlayerCount++;
-	}
+	internal virtual void OnClientJoined( IClient client ) { }
 
 	internal virtual void OnClientDisconnect( IClient cl, NetworkDisconnectionReason reason )
 	{
-		PlayerCount--;
+		if ( cl.Pawn is Player player )
+			DisconnectedPlayers.Add( player );

```

Comment:
Do we care about inline if statements vs. curly braces?

Reply:
What you did is fine, just don't put them on the same line.